contracts/HybridCustody.cdc

```diff
@@ -487,6 +491,13 @@ pub contract HybridCustody {
             return nil
         }
 
+        access(contract) fun removeSelfAsParent(parent: Address) {
+            let acct = self.childCap.borrow()!.borrowAccount()
+            if let m = acct.borrow<&ChildAccount>(from: HybridCustody.ChildStoragePath) {
```

Comment:
NP: Prefer a more descriptive variable name here, `m` isn't immediately clear to me

Reply:
Our go-like behavior is showing! 😅